administrate.gemspec

```diff
@@ -27,6 +27,8 @@ Gem::Specification.new do |s|
   s.add_dependency "sassc-rails", "~> 2.1"
   s.add_dependency "selectize-rails", "~> 0.6"
 
+  s.add_development_dependency "rspec-rails"
```

Comment:
Having this in the Gemfile wasn't playing well with the specific versions needed in the Appraisals. I was seeing an error about specifying the same gem twice.